src/core/hle/service/hid/hid.cpp

```diff
@@ -233,7 +189,7 @@ void Module::UpdateAccelerometerCallback(std::uintptr_t user_data, s64 cycles_la
     next_accelerometer_index = (next_accelerometer_index + 1) % mem->accelerometer.entries.size();
 
     Common::Vec3<float> accel;
-    std::tie(accel, std::ignore) = motion_device->GetStatus();
+    // std::tie(accel, std::ignore) = motion_device->GetStatus();
```

Comment:
Kraken should have all this data along with orientation. You can just grab what you need here from the struct